### PR TITLE
Logo

### PR DESCRIPTION
**Pull request by alycrys**
Made Logo compositional and created LogoImage and LogoSeparator components. 
I removed LogoTitle, can be replaced by Text components.

### CODE REVIEW

File: .changeset/nine-teachers-bow.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+"@salt-ds/lab": patch
+"docs": patch
+---
+
+Logo
+
+- Add LogoImage and LogoSeparator components
+- Remove LogoTitle
```
Comment (libertymayc) [inline .changeset/nine-teachers-bow.md:9]:
```suggestion
- Added LogoImage and LogoSeparator components
- Removed LogoTitle component
```

File: .changeset/nine-teachers-bow.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+"@salt-ds/lab": patch
+"docs": patch
```
Comment (libertymayc) [inline .changeset/nine-teachers-bow.md:3]: needed?

Comment (alycrys) [reply .changeset/nine-teachers-bow.md:3]: nope

File: packages/lab/stories/logo/logo.doc.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+### LogoImage Wrapped with Link
+
+<Canvas>
+  <Story id="lab-logo--link-on-image" />
+</Canvas>
+
+### Two Texts
+
+<Canvas>
+  <Story id="lab-logo--text-instead-image" />
```
Comment (libertymayc) [inline packages/lab/stories/logo/logo.doc.stories.mdx:59]: can this story be renamed to maybe Text Logo or Text As Logo

File: packages/lab/stories/logo/logo.doc.stories.mdx [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+### Link wrapping LogoImage only
+
+<Canvas>
+  <Story id="lab-logo--link-on-image" />
+</Canvas>
+
+### Compact Logo
+
+Here's an example of a reglular versus a compact version of Logo which now you can create yourself.
```
Comment (feliciackh) [inline packages/lab/stories/logo/logo.doc.stories.mdx:59]:
```suggestion
Here's an example of a regular versus a compact version of Logo which now you can create yourself.
```

File: packages/lab/stories/logo/logo.doc.stories.mdx [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+
+### Characteristics Used
+
+<CharacteristicUsage of={Logo} />
+
+## Accessibility
+
+Banner is accessible by assistive technologies like screen readers, and provides full WAI - ARIA support.
+
+Any focusable elements included in the Banner content will need to be checked and tested accordingly.
```
[the remark this answers is not in the document]
Comment (feliciackh) [reply packages/lab/stories/logo/logo.doc.stories.mdx:95]: probably not gonna need the second line on focusable elements within Logo?

**Review by feliciackh — APPROVED** [review]

**Review by feliciackh — APPROVED** [review]

**Pull request merged**